### PR TITLE
feat : history-page

### PR DESCRIPTION
**Pull request by mikesch803**
1. add history page with api integration

# files to check 

- [ ] src/components/stack-list-container/StackListContainer.jsx
- [ ] src/components/video-card/VideoCard.jsx
- [ ] src/context/history-context.jsx
- [ ] src/pages/history-page/History.jsx

## please login/signup first to check the deployed preview

### CODE REVIEW

File: src/pages/history-page/History.jsx [hunk 1]
```diff
@@ -1,7 +1,53 @@
...
+      try {
+        const response = await axios.get("/api/user/history", {
+          headers: {
+            authorization: encodedToken,
+          },
+        });
+        if (response.status === 200) {
+          setHistoryVideos(response.data.history);
+        }
+        console.log("getting history ", response);
```
Comment (ApurvaSawant11) [inline src/pages/history-page/History.jsx:24]: Remove this if not needed

File: src/pages/history-page/History.jsx [hunk 2]
```diff
@@ -1,7 +1,53 @@
...
+    <div className="history-video-grid-layout">
+      <Aside />
+      <main className="history-video-main">
+        <h2 className="history-video-title">
+          History Videos
+          <span
+            className="btn btn-ss btn-outline m-l-auto"
+            onClick={clearHistoryHandler}
+          >
+            clear
```
Comment (ApurvaSawant11) [inline src/pages/history-page/History.jsx:40]: Align this to the center.  It's on the right side of the page

**Review by ApurvaSawant11 — APPROVED** [review]
Everything works properly.  Did not find much to suggest.

Just a few suggestions
1. Add a `login as a guest` option. It helps. Not just for the reviews but anywhere you showcase your project. 
2. When there is only 1 video in history. The video card is shown on the right side. Try to fix that when you are done with basic functionalities.

**Review by Developers-codz — APPROVED** [review]
Everything looks good to me just one observation when you clear history `history video text ` and `button` is coming in the center. CSS properties for them are not working may be just check that

File: src/pages/history-page/History.jsx (hunk 1 above)
Comment (mikesch803) [reply src/pages/history-page/History.jsx:24]: okay, next time i will do the login guest btn/ share test credential

**Comment by mikesch803** [comment]
> Everything looks good to me just one observation when you clear history `history video text ` and `button` is coming in the center. CSS properties for them are not working may be just check that

yes css has some issue, but will do correct it

**Pull request merged**